docs/source/backends/gaudi.mdx

```diff
@@ -229,20 +171,8 @@ Note: Model warmup can take several minutes, especially for FP8 inference. For f
 #### Batch Size Parameters
 - For prefill operation, please set `--max-batch-prefill-tokens` as `bs * max-input-tokens`, where `bs` is your expected maximum prefill batch size.
 - For decode operation, please set `--max-batch-size` as `bs`, where `bs` is your expected maximum decode batch size.
-- Please note that batch size will be always padded to the nearest multiplication of `BATCH_BUCKET_SIZE` and `PREFILL_BATCH_BUCKET_SIZE`.
-
-#### Performance and Memory Parameters
-- `PAD_SEQUENCE_TO_MULTIPLE_OF` determines sizes of input length buckets. Since warmup creates several graphs for each bucket, it's important to adjust that value proportionally to input sequence length. Otherwise, some out of memory issues can be observed.
-- `ENABLE_HPU_GRAPH` enables HPU graphs usage, which is crucial for performance results. Recommended value to keep is `true`.
-
-#### Sequence Length Parameters
-- `--max-input-tokens`: Maximum possible input prompt length (default: 4095)
-- `--max-total-tokens`: Maximum possible total sequence length (input + output) (default: 4096)
+- Please note that batch size will be always padded to the nearest shapes what has been warmed up. This is done to avoid out of memory issues and to ensure that the graphs are reused efficiently.
```

Comment:
```suggestion
- Please note that batch size will be always padded to the nearest shapes that has been warmed up. This is done to avoid out of memory issues and to ensure that the graphs are reused efficiently.
```